podpac/core/data/datasource.py

```diff
@@ -281,6 +281,24 @@ def _get_data(self, rc, rci):
     # Methods
     # ------------------------------------------------------------------------------------------------------------------
 
+    def get_source_data(self, bounds={}):
+        """
+        Get source data, without interpolation.
+
+        Arguments
+        ---------
+        bounds : dict
+            Dictionary of bounds by dimension, optional.
+
+        Returns
+        -------
+        data : UnitsDataArray
+            Source data
+        """
+
+        coords, I = self.coordinates.select(bounds, return_index=True)
+        return self._get_data(coords, I)
```

Comment:
This is a great implementation. I think it's also very safe and general. I looked to see if any source over-writes `_get_data` and only the ogc & ogr sources do. The ogc data source should work fine, but the ogr one has a completely different method signature -- not sure if you want to handle that here...

Reply:
I was just walking the dogs and decided that I should implement `get_source_data` itself for the WCS node for this PR. You're right about the ogr node, too. In fact, it subclasses `Node`, not `Datasource`, so doesn't even have `get_source_data defined`.

Reply:
Well, it's not really possible for the ogc node. There is no native resolution.

Reply:
I think it's best to leave both ogr and ogc how they are. The only thing I might do is implement the ogc get_source_data in order to throw a more user-friendly exception.

Reply:
That makes sense. I didn't realize they didn't inherit from DataSource. Although, the OGC source sort-of has a native resolution which *may* be provided by the server.

Reply:
My mistake. The ogc node is fine: it does inherit from Datasource and we do pull the native resolution. It's not possible for the ogr node. That's the one that has no native coordinates.